src/libarpack.jl

```diff
@@ -73,26 +89,27 @@ function aupd_wrapper(T, matvecA!::Function, matvecB::Function, solveSI::Functio
                   iparam, ipntr, workd, workl, lworkl, info)
         end
         if info[] != 0
-            throw(ARPACKException(info[]))
+            throw(XYAUPD_Exception(info[]))
         end
 
         x = view(workd, ipntr[1] .+ zernm1)
         y = view(workd, ipntr[2] .+ zernm1)
         if mode == 1  # corresponds to dsdrv1, dndrv1 or zndrv1
-            if ido[] == 1
+            if ido[] == -1 || ido[] == 1
```

Comment:
This fixes #76 

cc @andreasnoack

Reply:
That was never the real issue there. It was just how it surfaced. See https://github.com/JuliaLinearAlgebra/Arpack.jl/issues/118